scripts/apps/web-publisher/directives/SiteRoutesDirective.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @ngdoc directive
+ * @module superdesk.apps.web_publisher
+ * @name sdSiteRoutes
+ * @requires publisher
+ * @description Directive to handle listing routes in web sites manager
+ */
+SiteRoutesDirective.$inject = ['publisher'];
+export function SiteRoutesDirective(publisher) {
+
+    class SiteRoutes {
+
+        constructor() {
+            this.scope = {site: '=site'};
+            this.templateUrl = 'scripts/apps/web-publisher/views/site-routes.html';
+        }
+
+        link(scope) {
+
+            scope.$on('refresh:routes', function(e, data) {
```

Comment:
I would document this event. Take a look [here](https://github.com/wingedfox/angular-route-segment/blob/master/src/route-segment.js#L679-L687) to see how events can be documented. Please check with `grunt docs` that it shows up.